src/components/VerticalStepper/VerticalStepper.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import VerticalStepper, { VerticalStepperProps } from "./VerticalStepper";
+const StepperExample = (props: VerticalStepperProps) => {
+  return (
+    <VerticalStepper {...props}>
+      <VerticalStepper.Step
+        label="Label 1"
+        collapsed={false}
+        status="complete"
+      >
+        Text Value 1
+      </VerticalStepper.Step>
+      <VerticalStepper.Step
+        label="Label 2"
+        status="complete"
+      >
+        Text Value 2
```

Comment:
this Text has a crazy font on the story. We should make it prettier. Maybe use our Text element?